app/src/main/java/com/android/sample/ui/createRecipe/RecipeListInstructionsScreen.kt

```diff
@@ -0,0 +1,270 @@
+        )
+      })
+}
+
+@Composable
+fun RecipeListInstructionsContent(
+    createRecipeViewModel: CreateRecipeViewModel,
+    modifier: Modifier,
+    navigationActions: NavigationActions,
+) {
+
+  Column(
+      modifier = modifier.padding(RECIPE_NAME_BASE_PADDING).testTag(SCREEN_COLUMN),
+  ) {
+    RecipeProgressBar(currentStep = CURRENT_STEP)
+    Spacer(
+        modifier = Modifier.height(BIG_PADDING.dp).testTag(RECIPE_LIST_INSTRUCTIONS_SCREEN_SPACER1))
+    Text(
+        modifier = Modifier.testTag(RECIPE_NAME_TEXT),
+        text = createRecipeViewModel.getRecipeName(),
+        style = MaterialTheme.typography.titleLarge,
+        color = MaterialTheme.colorScheme.onPrimary,
+    )
+    Text(
+        modifier = Modifier.testTag(INSTRUCTION_TEXT),
+        text = stringResource(R.string.RecipeListInstructionsScreen_Instructions),
+        style = MaterialTheme.typography.titleMedium,
+        color = MaterialTheme.colorScheme.onPrimary,
+    )
+    // LazyColumn for the scrollable list of instructions
+    LazyColumn(
+        contentPadding = PaddingValues(vertical = SMALL_PADDING.dp),
+        modifier =
+            Modifier.weight(1f)
+                .background(
+                    brush =
+                        Brush.verticalGradient(
+                            colors =
+                                listOf(MaterialTheme.colorScheme.background, Color.Transparent)))
+                .align(Alignment.CenterHorizontally)
+                // These parameters come from
+                // https://stackoverflow.com/questions/66762472/how-to-add-fading-edge-effect-to-android-jetpack-compose-column-or-row
+                .fadingEdge(
+                    Brush.verticalGradient(
+                        0f to Color.Transparent,
+                        0.05f to Color.Red,
+                        0.7f to Color.Red,
+                        1f to Color.Transparent))
+                .testTag(INSTRUCTION_LIST)) {
+
+          /**
+           * This Part will be later improved by adding the list of instructions
+           *
+           * items(createRecipeViewModel.getInstructions().length) { index ->
+           * InstructionValue(index, recipeBuilder.getTime(), 0,
+           * {navigationActions.navigateTo(Screen.CREATE_RECIPE_ADD_INSTRUCTION)}) }
+           */
+          item {
+            InstructionValue(
+                createRecipeViewModel = createRecipeViewModel,
+                index = 0,
+                time = createRecipeViewModel.getRecipeTime(),
+                onClick = { index ->
+                  createRecipeViewModel.selectDescription(index = index)
+                  navigationActions.navigateTo(Screen.CREATE_RECIPE_ADD_INSTRUCTION)
+                })
+          }
+        }
+
+    // Fixed button at the bottom
+    NextStepButton(
+        modifier = Modifier.align(Alignment.CenterHorizontally),
+        navigationActions = navigationActions,
+    )
+  }
+}
+
```

Comment:
Perhaps we should create a Composable in `ui/utils.kt` for this button, as it's used across multiple screens in the Create Recipe user flow. This would help with reusability and maintainability.

Reply:
Done :)